Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -3125,12 +3127,32 @@ def splunk_search_command(service: client.Service, args: dict) -> CommandResults
     return results
 
 
-def splunk_job_create_command(service: client.Service, args: dict):
+def splunk_job_create_command(service: client.Service, base_url: str, token: str, auth_token: str | None, args: dict):
     app = args.get("app", "")
     query = build_search_query(args)
     search_kwargs = {"exec_mode": "normal", "app": app}
     search_job = service.jobs.create(query, **search_kwargs)
 
+    share = args.get("share", "No")
+    if not share in ["Yes", "No"]:
+        share = "No"
+    if share == "Yes":
+        session_key = None if auth_token else token
+        headers = {"Authorization": f"Bearer {auth_token}"} if auth_token else {"Authorization": session_key}
+        save_results = requests.post(
```

Comment:
is there a way to do it through splunk SDK instead of http request, especially that the previous functionality was implemented via the SDK?